pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -102,7 +106,7 @@ type VitessClusterSpec struct {
 	Keyspaces []VitessKeyspaceTemplate `json:"keyspaces,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 
 	// ExtraVitessFlags can optionally be used to pass flags to all Vitess components.
-	// WARNING: Any flags passed here must be flags that can be accepted by vtgate, vtctld and vttablet.
+	// WARNING: Any flags passed here must be flags that can be accepted by vtgate, vtctld, orchestrator and vttablet.
```

Comment:
Is this actually true? If so this seems like a pretty big downside to this integration right now. Have we verified which flags orchestrator does not accept that vtgate, vtctld and vttablet does accept?

Reply:
I would think that if Orchestrator is independent of Vitess (is this orchestrator support or vorchestrator support?) that these flags should NOT be passed to orchestrator.

Reply:
This should generally be true. This version of Orchestrator is a vitess component. It shares (and requires) the same flags the other components do.

Reply:
Ahh, I see. So this is vorchestrator? If it's a Vitess component style of orchestrator, then we might want to clear this language up some. To me as a reader, it sounds like I need to go reference the main orchestrator code base to find what flags it accepts. 

In other words, this reads as if it is a breaking change, when it's not.